parachains/chain-specs/glutton.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Glutton",
+  "id": "glutton",
+  "chainType": "Live",
+  "bootNodes": [],
+  "telemetryEndpoints": null,
+  "protocolId": null,
+  "properties": null,
+  "relay_chain": "kusama",
+  "para_id": 2005,
```

Comment:
We can get a namespace between 1000 and 1999 for these paras. Maybe just put like 1300 here and then we can add new ones as 1301, 1302, ...